### PR TITLE
Add megalinter instructions to DEVELOPING.md and move settings to separate file

### PR DESCRIPTION
**Pull request by cfkoehler**

### CODE REVIEW

File: .mega-linter.yml
Comment (cfkoehler) [inline .mega-linter.yml:1]: - [ ] Still working to see if I can move this to the linters directory. But not sure how to set the action to use it when moved away from repo root

File: DEVELOPING.md [hunk 1]
```diff
@@ -293,6 +293,16 @@ additional things to consider when developing:
...
 * If you answer a question, add some documentation.
 
+### Local Mega Linter validation and auto fix
+The project leverages [MegaLinter](https://github.com/marketplace/actions/megalinter) as a GitHub action to run across 
+changed files in PR's and the entire codebase on merge. To run the same linting rule set local following the following steps.
+Instructions are derived from [MegaLinter Runner Documentation](https://megalinter.io/latest/mega-linter-runner/)
+
+1) Install [Node Package Manager (NPM)](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+2) Install mega-linter-runner for current users `npm install mega-linter-runner --save-dev`
+3) Run MegaLinter from project root using project configuration `mega-linter-runner  -e 'LINTER_RULES_PATH=.github/linters' .`
```
Comment (cfkoehler) [inline DEVELOPING.md:303]:
```suggestion
3) Run MegaLinter from project root using project configuration `mega-linter-runner  -e 'EXTENDS=.github/linters/.mega-linter.yml' -p .`
```

File: DEVELOPING.md [hunk 2]
```diff
@@ -293,6 +293,16 @@ additional things to consider when developing:
...
+* To have MegaLinter attempt to fix errors add the `--fix` option to the command: `mega-linter-runner  -e 'LINTER_RULES_PATH=.github/linters' --fix . `
```
Comment (cfkoehler) [inline DEVELOPING.md:304]:
```suggestion
* To have MegaLinter attempt to fix errors add the `--fix` option to the command: `mega-linter-runner  -e 'EXTENDS=.github/linters/.mega-linter.yml' --fix -p .`
```

**Review by jpdahlke — DISMISSED** [review]

**Review by Kevinnnnnnnnnnnnnnnn — CHANGES_REQUESTED** [review]
Other than some language I found confusing in DEVELOPING.md, everything looks good to me!

File: DEVELOPING.md [hunk 3]
```diff
@@ -293,6 +293,16 @@ additional things to consider when developing:
 * If you fix a bug, add a test.
 * If you answer a question, add some documentation.
 
+### Local Mega Linter validation and auto fix
+The project leverages [MegaLinter](https://github.com/marketplace/actions/megalinter) as a GitHub action to run across 
+changed files in PR's and the entire codebase on merge. To run the same linting rule set local following the following steps.
```
Comment (cfkoehler) [inline DEVELOPING.md:298]:
```suggestion
changed files in PR's and the entire codebase on merge. To run the same linting rule set locally, follow the steps bellow:
```

File: DEVELOPING.md [hunk 4]
```diff
@@ -293,6 +293,16 @@ additional things to consider when developing:
 * If you fix a bug, add a test.
 * If you answer a question, add some documentation.
 
+### Local Mega Linter validation and auto fix
+The project leverages [MegaLinter](https://github.com/marketplace/actions/megalinter) as a GitHub action to run across 
+changed files in PR's and the entire codebase on merge. To run the same linting rule set locally, follow the steps bellow:
```
Comment (fbruton) [inline DEVELOPING.md:298]:
```suggestion
changed files in PR's and the entire code-base on merge. To run the same linting rule set locally, follow the steps below:
```

**Review by fbruton — DISMISSED** [review]
Fix minor typo

**Review by jpdahlke — APPROVED** [review]

**Review by Kevinnnnnnnnnnnnnnnn — APPROVED** [review]

**Pull request merged**